download.py

```diff
@@ -0,0 +1,153 @@
+from pathlib import Path
+from downloader.timeseries_downloader import TimeseriesDownloader
+from downloader.yahoo_downloader import YahooDownloader
+from downloader.alphavantage_downloader import AVDownloader
+from typing import List, Dict
+from datetime import date, datetime, timedelta
+from config import Config
+import json
+
+DATA_FOLDER = Path("data")
+DOWNLOADER_NAMES = ["YahooFinance", "AlphaVantage"]
+DOWNLOADER_CLASS: List[TimeseriesDownloader] = [
+    YahooDownloader(), AVDownloader(Config.get_config("alphavantage_api_key"))]
+
+TICKER_LISTS_FOLDER = Path("downloader/docs")
+
+
+def check_and_create_folder(path: Path):
+    '''
+    Creates data folder if it doesn't exist
+    '''
+    if(not path.exists()):
+        path.mkdir(exist_ok=True)
+    return path
+
+
+def choose_downloader() -> int:
+    '''
+    Choose which downloader to use from the available ones.
+
+    Returns:
+        The integer index of the selected downloader.
+    '''
+    while(True):
+        choice = input("Choose between: {} ".format(DOWNLOADER_NAMES))
+        if(choice in DOWNLOADER_NAMES):
+            break
+        print("Unable to parse ", choice)
+    return DOWNLOADER_NAMES.index(choice)
```

Comment:
Mind if I ask why not just a dictionary?

Reply:
Because I'm probably dumb